src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
```

Comment:
I disagree, I'd rather pay for having one extra function once over having every machine that runs our software pay for an extra move over and over again to infinity.